webui/react/src/components/kit/LogViewer/LogViewer.tsx

```diff
@@ -578,7 +579,7 @@ const LogViewer: React.FC<Props> = ({
         <div className={css.container}>
           <div className={css.logs} ref={logsRef}>
             <VariableSizeList
-              height={containerSize.height}
```

Comment:
Why are you changing these other components?

Reply:
Without this change, the LogView would look like folded. 
<img width="1628" alt="Screen Shot 2023-05-03 at 4 06 56 PM" src="https://user-images.githubusercontent.com/40620519/236050171-32ba5067-c15c-4cb6-991b-eb7bd380cd86.png">

Reply:
Ah, this is because of your change to Pivot?

Reply:
Not sure this answers your question. Currently we have tab's height at 100%, so the LogViewer can fill up that space. If we change the tab's height to auto, LogViewer gets folded if not specifying a height